src/main.c

```diff
@@ -1114,6 +1114,9 @@ main(int argc, char **argv)
   signal(SIGPIPE, handle_sigpipe); // will be redundant later
   signal(SIGILL, handle_sigill);   // see handler..
 
+  if (opt_fork && !opt_user && !opt_config)
+    tvhwarn(LS_START, "Forking without --user or --config may use unexpected configuration location");
```

Comment:
I'd try to use wording that's more instructional than informational, e.g. "Fork with --user and --config to avoid configuration storage in system folders"

Reply:
Hey Chris, missed your comment and of course not a native english speaker like yourself :) As the MR was open since august, I kind of lost track of it :)